tests/e2e/Makefile

```diff
@@ -33,8 +33,12 @@ help: ## Display this help.
 
 .PHONY: create_module_kubebuilder_project
 create_module_kubebuilder_project:
-	go test --tags=create_module_kubebuilder_project
+	go test ./create_module -run "Test_ModuleTemplate"
 
 .PHONY: create_module_module_config
 create_module_module_config:
-	go test --tags=create_module_module_config
+	go test ./create_module -run "Test_ModuleTemplate"
+
+.PHONY: deploy_enable_disable
+deploy_enable_disable:
```

Comment:
I advocate for sticking to the kebab-case in the targets naming instead of a snake case (we use it in the Makefile in the root of the repository and in the LM repository too).